Sources/Sentry/SentryOptions.m

```diff
@@ -30,11 +30,23 @@ - (instancetype)init
         self.attachStacktrace = YES;
         self.maxAttachmentSize = 20 * 1024 * 1024;
         self.sendDefaultPii = NO;
+
+        // Use the name of bundle’s executable file as inAppInclude, so frames coming from there are
+        // marked as inApp. This also marks private frameworks included in the application
+        // statically as inApp. Sentry fixes this with stack trace grouping rules.
```

Comment:
M: does the phrase `Sentry fixes this with stack trace grouping rules` mean that the SDK's code mark frames as inApp, but the grouping rules on the server will make them not inApp? if so, I'd suggest rewriting this comment and being a bit more clearer.

Reply:
Clearer now @marandaneto?